Dockerfile

```diff
@@ -12,15 +10,14 @@ RUN apt-get update -y && apt-get install -y --no-install-recommends libclang-dev
 
 ARG BINDGEN_EXTRA_CLANG_ARGS="-I/usr/include/scotch"
 
-RUN cargo install --path tools --root /builder/install
+RUN cargo install --path tools --root /builder/install --features metis,intel-perf
```

Comment:
Does this thing work on non x86 cpus ?

Reply:
If I remember correctly building with ittapi wasn't a problem even on AMD cpus, but I haven't tested with other instruction sets.